### ISSUE TITLE
Add in translation support everywhere we can

### ISSUE THREAD

**Issue by rogerhutchings** [zooniverse/front-end-monorepo#407]
**Feature or Issue Description**

At the moment, we don't need translations for TESS, so in the interests of hitting our deadline I think we should punt it for now. But we should add in proper translation support as soon as feasible, since as @eatyourgreens rightly pointed out, adding it in everywhere after the fact will be such pain

cc #54

**Comment by eatyourgreens** [comment]
I'm thinking that wherever we have JSX code like
```jsx
<p>
  {tutorial.steps[2].content}
</p>
```
we should have a simple component that takes a dictionary key and renders the string for a given locale
```jsx
<LanguageString
  type="tutorial"
  id="123"
  key="steps.2.content"
  locale="en"
/>
```
and it will be much less work in the long run if we localise those strings when we first build components.

The pain point in localising PFE was searching through the code trying to find every rendered language string, and I'm still not confident that I found everything that needs to be localised.

**Comment by srallen** [comment]
Could that be something added to the react-components lib?

**Comment by srallen** [comment]
Sort of a duplicate of #54

**Comment by eatyourgreens** [comment]
For now, `TODO:` comments should work wherever we need to go back and rewrite code.

**Comment by rogerhutchings** [comment]
cc #54

**Comment by rogerhutchings** [comment]
Time estimate: 6 weeks, just to make sure we've caught every string that needs localising

**Comment by rogerhutchings** [comment]
Closed in favour of #54
